Roslyn.runsettings

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>  
+<RunSettings>  
+  <RunConfiguration>  
+    <TestAdaptersPaths>%userprofile%\.nuget\packages\xunit.runner.visualstudio\2.2.0-beta4-build1194</TestAdaptersPaths>  
```

Comment:
This is due to https://github.com/dotnet/project-system/issues/2129.  We *should* be able to just pick up the adapter from the nuget references, but the fact that it uses a property instead of a hard-coded value means that the nuget APIs won't return it to us.  Unfortunately, this is the only way I could find to work around the issue.  We ***could*** circumvent the nuget APIs and inspect the PackageReferences ourselves or add a LUT specific project property (something like ```<LiveUnitTestingTestAdapterPaths>```), but I've been resisting adding yet another way to include adapters.